src/post0.12.0/nodehun.cpp

```diff
@@ -4,14 +4,14 @@ using namespace v8;
 
 Persistent<Function> Nodehun::SpellDictionary::constructor;
 
-void Nodehun::SpellDictionary::Init(Handle<Object> exports, Handle<Object> module) 
+void Nodehun::SpellDictionary::Init(Handle<Object> exports, Handle<Object> module)
 {
   Isolate* isolate = Isolate::GetCurrent();
   Local<FunctionTemplate> tpl = FunctionTemplate::New(isolate, New);
   tpl->SetClassName(String::NewFromUtf8(isolate, "NodehunDictionary"));
   tpl->InstanceTemplate()->SetInternalFieldCount(1);
   //static
-  NODE_SET_METHOD(tpl, "createNewNodehun" , createNewNodehun);    
+  NODE_SET_METHOD((v8::Local<v8::Template>) tpl, "createNewNodehun" , createNewNodehun);    
```

Comment:
Please find the latest node headers [here](https://github.com/nodejs/node/blob/master/src/node.h).

The latest version of node ships with headers which overload `NODE_SET_METHOD` (see lines 238 and 251). Thus, the coercion was necessary in order to disambiguate between the two method signatures.